bats/core/api/csv.bats

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bats
+
+load "../../helpers/callback.bash"
+load "../../helpers/cli.bash"
+load "../../helpers/ledger.bash"
+load "../../helpers/ln.bash"
+load "../../helpers/onchain.bash"
+load "../../helpers/subscriber.bash"
+load "../../helpers/trigger.bash"
+load "../../helpers/user.bash"
+
+ALICE='alice'
+
+setup_file() {
+  create_user "$ALICE"
+  add_callback "$ALICE"
+  fund_user_onchain "$ALICE" 'btc_wallet'
+}
+
+btc_amount=1000
+usd_amount=50
+
+@test "ln-receive: settle via ln for USD wallet, invoice with amount" {
+  # Generate invoice
+  token_name="$ALICE"
+  usd_wallet_name="$token_name.usd_wallet_id"
+
+  variables=$(
+    jq -n \
+    --arg wallet_id "$(read_value $usd_wallet_name)" \
+    --arg amount "$usd_amount" \
+    '{input: {walletId: $wallet_id, amount: $amount}}'
+  )
+  exec_graphql "$token_name" 'ln-usd-invoice-create' "$variables"
+  invoice="$(graphql_output '.data.lnUsdInvoiceCreate.invoice')"
+
+  payment_request="$(echo $invoice | jq -r '.paymentRequest')"
+  [[ "${payment_request}" != "null" ]] || exit 1
+  payment_hash="$(echo $invoice | jq -r '.paymentHash')"
+  [[ "${payment_hash}" != "null" ]] || exit 1
+
+  # Receive payment
+  lnd_outside_cli payinvoice -f \
+    --pay_req "$payment_request"
+
+  # Check for settled
+  retry 15 1 check_for_ln_initiated_settled "$token_name" "$payment_hash"
+}
```

Comment:
if this is required for the test please move the content to the proper test... otherwise looks like it belongs to ln-receive use cases

Reply:
what would you do? move the `testing csv` within ln-receive use case?

I need to have some transactions to test that the csv return something.